### DISCUSSION TITLE
How to make file explorer look newer

### DISCUSSION THREAD

**Discussion by lndfebfdjo** [The-Wineskin-Project/WineskinServer#292] [Q&A] [answered]
Is there a way to make the file explorer and other apps look newer. If possible i'd like to make them look like windows 10, or at least like Windows 8. 

Right now it looks like this:
<img width="1680" alt="Screenshot 2022-08-06 at 18 24 31" src="https://user-images.githubusercontent.com/92981031/183255647-13948e13-180e-4f06-b13a-c5011aec8259.png">
<img width="1680" alt="Screenshot 2022-08-06 at 18 23 20" src="https://user-images.githubusercontent.com/92981031/183255681-93dfac16-01ee-49cc-972f-9724c6df9ea8.png">

**Comment by emendelson** [comment, marked as answer]
This is the explorer.exe created by the upstream Wine project. You can ask for an enhancement at

https://winehq.org 

and anything that gets changed will eventually get into Wineskin.
